_topic_maps/_topic_map.yml

```diff
@@ -2013,7 +2013,7 @@ Topics:
   File: uninstalling-sandboxed-containers
 - Name: Upgrade OpenShift sandboxed containers
   File: upgrade-sandboxed-containers
-- Name: Collecting OpenShift sandboxed containers data for Red Hat Support
+- Name: Collecting OpenShift sandboxed containers data
```

Comment:
Hi @MirzWeiss, not directly connected to this commit, but I see this in the rendered file:

```
$ oc adm must-gather  --image=registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8:v4.10.0
```

Is this intentional? Seeing `cnv` in the documentation about sandboxed containers makes me suspicious.

Reply:
Hi @c3d - this is located in a shared module and is not specific to only OSC. The image you're asking about it located in an example. At the end of this section I added the exact image our users need to use.

Reply:
@MirzWeiss Thanks. The rest LGTM.